docs/how-to/code/install-slice/task.yaml

```diff
@@ -17,12 +21,7 @@ execute: |
   grep -q "bins" chisel-releases/slices/openssl.yaml && echo "My slice exists"
   # [docs:slice-exists-end]
 
-  git clone https://github.com/canonical/chisel
-  pushd chisel
-  git reset --hard f0bff5a30dfdcb400b3c4efe85962ad8ff3ca2ba
-  go build ./cmd/chisel
-  mv chisel /bin/chisel
-  popd
+  sudo snap install chisel --channel=latest/candidate
```

Comment:
> mv rockcraft.yaml.backup rockcraft.yaml

You don't need this error-throwing line anymore because `rockcraft.yaml` isn't there.

Reply:
true, I'm bringing it to the beginning (we still need that file)

Reply:
see https://github.com/canonical/rockcraft/pull/456/commits/926ff933a1dbff61287c95620612c7f84da97cbc